### ISSUE TITLE
Integration with Ansible

### ISSUE THREAD

**Issue by hbbio** [wallix/awless#25]
awless should integrate with configuration management software. For instance, we could easily start an ansible "central" server and automatically add newly created instances to the ansible `hosts`.

**Comment by luisdavim** [comment]
or awlwss coyld be used as a dynamic host inventory, maybe this could be an output format option.

**Comment by simcap** [comment]
Indeed. One of the first step is to provide a dynamic host inventory with `awless`. 

`awless` will therefore provide the JSON output filling in inventory values. The question is more around the format, grouping and content of the JSON file.

@luisdavim Any input / ideas on that is welcomed!

Documentation: 
* http://docs.ansible.com/ansible/dev_guide/developing_inventory.html
* http://docs.ansible.com/ansible/intro_dynamic_inventory.html
* http://docs.ansible.com/ansible/intro_inventory.html

**Comment by luisdavim** [comment]
This is a good example: https://github.com/ansible/ansible/blob/devel/contrib/inventory/ec2.py

Another option is to have a ini output format and use that as a static inventory generator.

**Comment by luisdavim** [comment]
Or we can always use all from the ansible perspective and do the filtering on awless.
